turtle_tf2_py/turtle_tf2_py/turtle_tf2_message_broadcaster.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from geometry_msgs.msg import PointStamped
+from geometry_msgs.msg import Twist
+
+import rclpy
+from rclpy.node import Node
+
+from turtlesim.msg import Pose
+from turtlesim.srv import Spawn
+
+
+class PointPublisher(Node):
+
+    def __init__(self):
+        super().__init__('turtle_tf2_message_broadcaster')
+
+        # Create a client to spawn a turtle
+        self.spawner = self.create_client(Spawn, 'spawn')
+        # Boolean values to store the information
+        # if the service for spawning turtle is available
+        self.turtle_spawning_service_ready = False
+        # if the turtle was successfully spawned
+        self.turtle_spawned = False
+        # if the topics of turtle3 can be subscribed
+        self.turtle_pose_cansubscribe = False
+        
+        self.timer = self.create_timer(1.0, self.on_timer)
+
+    def on_timer(self):   
```

Comment:
Can you take at the build test output?
```
  E     ./turtle_tf2_py/turtle_tf2_message_broadcaster.py:39:1: W293 blank line contains whitespace
  E     ./turtle_tf2_py/turtle_tf2_message_broadcaster.py:42:24: W291 trailing whitespace
```

Reply:
@kurshakuz Sorry, I checked carefully for all the files to ensure there is no whitespace ,but I still neglected these two lines. I removed them just now.